### ISSUE TITLE
fatal error: 'mpv/client.h' file not found

### ISSUE THREAD

**Issue by c1ngular** [aynakeya/go-mpv#2]
i use hombrew to install mpv , give me error as following :

# github.com/aynakeya/go-mpv
  ../../pkg/mod/github.com/aynakeya/go-mpv@v0.0.7/error.go:4:10: fatal error: 'mpv/client.h' file not found
  #include <mpv/client.h>
           ^~~~~~~~~~~~~~
  1 error generated.
  
  exit status 1

**Comment by aynakeya** [comment]
Hi!
Can you check if brew has installed the mpv header files? 

also, it is possible that brew install header files in a different location.

in that case. you need to manually set CFLAGS

related links: [link1](https://apple.stackexchange.com/questions/40704/homebrew-installed-libraries-how-do-i-use-them), https://github.com/AynaLivePlayer/AynaLivePlayer/issues/4

**Comment by c1ngular** [comment]
@aynakeya 

still complains : 

`          /usr/local/go/pkg/tool/darwin_arm64/link: running clang failed: exit status 1
          ld: library not found for -lmpv
          clang: error: linker command failed with exit code 1 (use -v to see invocation)`

after i set the ENV : 

export GO_LDFLAGS="-L/opt/homebrew/Cellar/mpv/0.35.1_2/lib"
export CGO_CFLAGS="-I/opt/homebrew/Cellar/mpv/0.35.1_2/include"

quick question : why not use pkg-config ? it is already shipped with . 

thanks

**Comment by aynakeya** [comment]
Yes, you can indeed use pkg-config to retrieve the necessary CFLAGS and library paths. The reason I didn't  using pkg-config  is that Windows systems typically do not have pkg-config preinstalled.

for you problem. can you double check if those flag appear in `go env`

![Screenshot_20230626_095147](https://github.com/fyne-io/fyne/assets/32156054/434a38aa-e9ea-483b-bd10-6c5cb79f4b2a)
